### ISSUE TITLE
compilation warnings

### ISSUE THREAD

**Issue by simonkramer** [jcodec/jcodec#99]
Hi,

Very useful library - thank you for making it available :-)

Please consider relevant compilation warnings in the attached file:

[jcodec-warnings.txt](https://github.com/jcodec/jcodec/files/44868/jcodec-warnings.txt)

It would be great if you could update the library and eliminate the warnings.

Many thanks,

Simon

**Comment by svitvitskiy** [comment]
Yes, I am half way into removing int array API in favour of byte array API.
All int array based classed are marked as deprecated however it will take
time to refactor all the available codecs. Thats why deprecation warnings.

Stan
